app.js

```diff
@@ -5,17 +5,19 @@ const anchor = document.querySelector("main");
 function clearAnchor() {
   anchor.innerHTML = "";
 }
-
+// function that collect data on json file
 function collectScenarios() {
   const json = fetch("./stories.json").then((response) =>
     response.json().then((data) => data.map((data) => data))
   );
   return {
     json,
-    renderInputs: (e) => console.log("clicked", e),
+    renderInputs: (json, e) => {
+      return console.log(json[e].Scenario_title, json[e].Variables, e);
```

Comment:
This should not return a console.log but the actual value but we needed this to confirm it works. I think we can leave it and let Mili change to call his function after he makes it.

Reply:
okay, @mnichols08 ... Thanks for reviewing it!

Reply:
It is a little bit redundant as we are passing json to the function and then back to this function rather than sending the actual story we are clicking on, but it works so we can push it to development.